lib/maze/client/appium/base_client.rb

```diff
@@ -58,6 +58,7 @@ def start_driver(config)
                   end
                   result
                 rescue => start_error
+                  $logger.error "Session creation failed: #{start_error}"
```

Comment:
Bit of an aside - I messed up during the development of this and it was falling in a heap but giving no feedback to me.